<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPatientWindow</class>
 <widget class="QWidget" name="addPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Please fill out the form below for the patient's information.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPatient">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>189</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ailment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>59</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AilmentButton">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>215</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add more ailments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>137</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Treated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>163</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>111</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Suffix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>85</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>33</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>242</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Doctor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>33</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="middleNameEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>59</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>85</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="suffixEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>111</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="treatedEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>137</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="priorityEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>163</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ailmentEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>189</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorEdit">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>242</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>AddPatient</zorder>
  <zorder>Cancel</zorder>
  <zorder>firstNameEdit</zorder>
  <zorder>middleNameEdit</zorder>
  <zorder>lastNameEdit</zorder>
  <zorder>suffixEdit</zorder>
  <zorder>treatedEdit</zorder>
  <zorder>priorityEdit</zorder>
  <zorder>ailmentEdit</zorder>
  <zorder>doctorEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
